src/components/form/form.scss

```diff
@@ -10,12 +10,8 @@
     }
 }
 
-.limel-form-layout--grid {
+.limel-form-layout--default {
+    --gap: 1rem;
     display: grid;
-    grid-template-columns: repeat(auto-fill, minmax(18.75rem, 1fr));
-    gap: 0.5rem;
```

Comment:
Except for the change from 0.5rem to 1rem gap, I think this commit is more of a refactor than a feature… 🤔

(And except for the possibility for the consumer to change `--gap`, if that's now possible.)

Reply:
I have no objections.